programs/survival/README.md

```diff
@@ -248,8 +247,9 @@ Various scripts that modify various game elements, often replicating popular mod
 
 ### villager_poi.sc:
 #### By gnembonmc
-	Keeps track of villager poi system?
-	(Please enter an explanation, as it quite unclear)
+	Keeps track of villager poi system
+	It allows the player to see which bed the villager is linked to by clicking the villager with an emerald.
+	It also shows what beds the villager could possibly link to as well as which beds it cannot.
```

Comment:
Wasn't there already a PR adding this that was rejected since this is explaining `/track` command instead of the app in here? (there isn't any single reference to emeralds in that app).

Reply:
ye...

Reply:
yeah sorry about that i forgot that my for had those prior changes